src/entrypoints/content/content.scss

```
@@ -53,19 +37,62 @@ body {
     outline: 1px solid #DDD;
   }
 
-  .hidden {
-    display: none;
+  .splitter {
+    // visually a 1px line; the wider grab area is the ::before overlay below
+    flex: 0 0 1px;
+    height: 100%;
+    position: relative;
+    background: #DDD;
+    // sit above the iframes so the grab overlay catches the cursor over them
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)